src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_completers.py

```diff
@@ -0,0 +1,26 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands.parameters import get_one_of_subscription_locations
+from azure.cli.core.decorators import Completer
+
+from ._client_factory import cf_compute_service
+
+
+@Completer
+def get_vm_size_completion_list(cmd, prefix, namespace, **kwargs):  # pylint: disable=unused-argument
+    print(prefix)
+    print(namespace)
+    raise NotImplementedError()
```

Comment:
Why is this here?

Reply:
Oops, some debug code I failed to remove.